pkg/groupsync/manytomanysyncer_test.go

```diff
@@ -1385,7 +1387,9 @@ func (tc *testReadWriteGroupClient) GetMembers(ctx context.Context, groupID stri
 	if err, ok := tc.getMembersErrs[groupID]; ok {
 		return nil, err
 	}
+	tc.mutex.RLock()
```

Comment:
Since it's test, I think we can be more relaxed where we locl/unlock. For simplicity, I'd just have these at the beginning of the func. Same for the other function.

```suggestion
	tc.mutex.RLock()
	defer tc.mutex.RUnlock()
```

Reply:
done.